src/node_api.cc

```diff
@@ -772,6 +772,11 @@ napi_status napi_get_last_error_info(napi_env env,
   return napi_ok;
 }
 
+NAPI_NO_RETURN void napi_fatal_error(const char* location,
+                                     const char* message) {
+  node::FatalError(location, message);
```

Comment:
Based on discussion today in the WG meeting, we'll be reverting back to the original change and removing the duplicated code.